packages/SwingSet/src/controller.js

```diff
@@ -205,33 +201,66 @@ export function loadSwingsetConfigFile(configPath) {
   }
 }
 
+/**
+ * Build the kernel source bundles.
+ *
+ */
+export async function buildKernelBundles() {
+  // this takes 2.7s on my computer
+  const sources = {
+    kernel: require.resolve('./kernel/kernel.js'),
+    adminDevice: require.resolve('./kernel/vatAdmin/vatAdmin-src'),
+    adminVat: require.resolve('./kernel/vatAdmin/vatAdminWrapper'),
+    comms: require.resolve('./vats/comms'),
+    vattp: require.resolve('./vats/vat-tp'),
+    timer: require.resolve('./vats/vat-timerWrapper'),
+  };
```

Comment:
Not obviously a use case we have an immediate need to support, but it strikes me that we've got most of the building blocks here to allow kernels to be built without some of this stuff in cases where the swingset being launched doesn't need them.

Reply:
yeah, #1351 is an opportunity to set up devices a bit more coherently, in which case we could defer adding the timer vat until someone gives us a timer device to work with. OTOH #1346 might manage to make devices sufficiently vat-like that we don't need wrappers anymore and that one goes away.

I'm inclined to think that comms is a sufficiently core part of the swingset experience ("what's life without IO?") that it's probably always going to be there, but yeah, who knows.